From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: celenity <celenity@celenity.dev>
Date: Sun, 05 Jul 2026 20:19:20 +0000
Subject: [PATCH] Gecko - IronFox branding

Modifies branding elements to IronFox.

(Parts of this have been adapted from LibreWolf: https://librewolf.dev/librewolf/source/src/branch/main/patches/ui-patches/neterror.patch)

Signed-off-by: celenity <celenity@celenity.dev>
---
diff --git a/toolkit/content/aboutSupport.js b/toolkit/content/aboutSupport.js
index 58de33cd7d60c..5235710277c55 100644
--- a/toolkit/content/aboutSupport.js
+++ b/toolkit/content/aboutSupport.js
@@ -13,6 +13,9 @@ const { ResetProfile } = ChromeUtils.importESModule(
 const { AppConstants } = ChromeUtils.importESModule(
   "resource://gre/modules/AppConstants.sys.mjs"
 );
+const { IFConstants } = ChromeUtils.importESModule(
+  "resource://gre/modules/IFConstants.sys.mjs"
+);
 
 ChromeUtils.defineESModuleGetters(this, {
   DownloadUtils: "resource://gre/modules/DownloadUtils.sys.mjs",
@@ -111,7 +114,7 @@ var snapshotFormatters = {
       "XREExeF",
       Ci.nsIFile
     ).path;
-    $("supportLink").href = data.supportURL;
+    $("supportLink").href = IFConstants.IRONFOX_FAQ_URL;
     let version = AppConstants.MOZ_APP_VERSION_DISPLAY;
     if (data.vendor) {
       version += " (" + data.vendor + ")";
diff --git a/toolkit/themes/shared/aboutNetError.css b/toolkit/themes/shared/aboutNetError.css
index b345001410b85..4069ff0b5f833 100644
--- a/toolkit/themes/shared/aboutNetError.css
+++ b/toolkit/themes/shared/aboutNetError.css
@@ -298,7 +298,7 @@ button:disabled {
   }
 
   .img-container > img {
-    width: 8.75em;
+    width: 3.5em;
     margin-block-start: 2em;
     margin-inline-end: 2.5em;
 
-- 
